app/modules/collaborations/parameters.py

```diff
@@ -6,12 +6,24 @@
 
 # from flask_marshmallow import base_fields
 from flask_restx_patched import Parameters, PatchJSONParameters
+from flask_marshmallow import base_fields
 
 from . import schemas
 from .models import Collaboration
 
 
 class CreateCollaborationParameters(Parameters, schemas.DetailedCollaborationSchema):
+    user_guid = base_fields.UUID(
```

Comment:
Suggestions of better names please. 
Used for both 
1 when  a researcher initiates a collaboration to another user (user_guid used but not second_user_guid) 
2 when a user manager creates a collaboratioon between two researchers